migration/migration_trello_to_mongo.py

```diff
@@ -0,0 +1,66 @@
+# Migration script - from trello to mongo
+# get data from trello collection
+# insert into mongo db - if it hasnt already been inserted
+import os
+
+# Trello
+import jsons
+from dotenv import load_dotenv
+
+from migration.trello_request import *
+from todoapp.common import Lists
+from todoapp.mongo_database import MongoDatabase
+
+# Get the path to the directory this file is in
+BASEDIR = os.path.abspath(os.path.dirname(__file__))
+ENV_FULL_PATH = os.path.join(BASEDIR, '.env')
+print(f"ENV_FULL_PATH: {ENV_FULL_PATH}")
```

Comment:
Is there a reason you've chosen to do this yourself, rather than relying on the `find_dotenv` method used elsewhere in your testing?